File: 2025-1/sesion-de-clase-20/SoftProgNet/SoftProgModelo/Logistica/Almacen/Producto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PUCP.Edu.Pe.SoftProg.Modelo.Logistica.Almacen {
    public class Producto : ModeloBase {
        public string Nombre { get; set; }
        public string UnidadMedida { get; set; }
        public double Precio { get; set; }
    }
}

File: 2025-1/sesion-de-clase-20/SoftProg/SoftProgModelo/src/main/java/pe/edu/pucp/inf30/softprog/model/logistica/almacen/Producto.java
package pe.edu.pucp.inf30.softprog.model.logistica.almacen;

import pe.edu.pucp.inf30.softprog.model.ModeloBase;

/**
 *
 * @author eric
 */
public class Producto extends ModeloBase {
    private String nombre;
    private String unidadMedida;
    private double precio;
    
    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getUnidadMedida() {
        return unidadMedida;
    }

    public void setUnidadMedida(String unidadMedida) {
        this.unidadMedida = unidadMedida;
    }

    public double getPrecio() {
        return precio;
    }

    public void setPrecio(double precio) {
        this.precio = precio;
    }
}
